sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -87,42 +99,42 @@ function createTagsFromSpan(span: ReadableSpan): Tags {
   }
   // TODO: Operation Name and Location IP TBD for non server spans
 
-  const httpUserAgent = span.attributes[SemanticAttributes.HTTP_USER_AGENT];
-  if (httpUserAgent) {
-    // TODO: Not exposed in Swagger, need to update def
-    tags["ai.user.userAgent"] = String(httpUserAgent);
-  }
-
   return tags;
 }
 
-function createPropertiesFromSpan(span: ReadableSpan): [Properties, Measurements] {
-  const properties: Properties = {};
-  const measurements: Measurements = {};
-
-  for (const key of Object.keys(span.attributes)) {
-    if (
-      !(
-        key.startsWith("http.") ||
-        key.startsWith("rpc.") ||
-        key.startsWith("db.") ||
-        key.startsWith("peer.") ||
-        key.startsWith("net.")
-      )
-    ) {
-      properties[key] = span.attributes[key] as string;
+function createPropertiesFromSpanAttributes(attributes?: SpanAttributes): {
```

Comment:
Might want to rename this `createPropertiesFromAttributes` and change the typing of `attributes` to `dict`? Since `event.attributes` are not `SpanAttributes`.

Reply:
If types changed, I would handle it in different PR, we need to update dependencies to latest and typescript build will fail if type is incorrect, updating dependencies usually require updating mono repo files and is hard to keep it up to date, so I prefer to make those changes separately.